src/main/webapp/app/exercises/shared/dashboards/tutor/exercise-assessment-dashboard.component.ts

```diff
@@ -340,39 +370,70 @@ export class ExerciseAssessmentDashboardComponent implements OnInit, AfterViewIn
     };
 
     /**
-     * Get a submission from the server that does not have an assessment yet (if there is one). The submission gets added to the end of the list of submissions in the exercise
-     * dashboard and the user can start the assessment. Note, that the number of started but unfinished assessments is limited per user and course. If the user reached this limit,
+     * Get all submissions that dont have an assessment for all correctionrounds
+     * If not in examMode correctionrounds defaults to 1.
```

Comment:
defaults to 0, as you use `0` as the parameter?

Reply:
you are right.